src/TLuaInterpreter.cpp

```diff
@@ -7942,11 +7943,70 @@ int TLuaInterpreter::tempAlias(lua_State* L)
     return 1;
 }
 
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#findItems
+int TLuaInterpreter::findItems(lua_State* L)
+{
+    int n = lua_gettop(L);
+    auto name = getVerifiedString(L, __func__, 1, "item name");
+    // Although we only use 6 ASCII strings the user may not enter a purely
+    // ASCII value which we might have to report...
+    QString type = getVerifiedString(L, __func__, 2, "item type").toLower();
+    bool exactMatch = true;
+    bool caseSensitive = true;
+    if (n > 2) {
+        exactMatch = getVerifiedBool(L, __func__, 3, "exact match", true);
+    }
+    if (n > 3) {
+        caseSensitive = getVerifiedBool(L, __func__, 3, "case sensitive", true);
+    }
+    Host& host = getHostFromLua(L);
+    auto generateList = [](auto vector, auto l) {
+        lua_newtable(l);
+        int index = 0;
+        for (const auto& item : vector) {
```

Comment:
qAsConst() not needed here?

Reply:
I've made `vector` a `const` in the declaration (on line 7963) in af69f2ba24cf378d5a81995745653f6ee88cad3f so it is definitely not needed now!